### ISSUE TITLE
Versioning of SAPUI5: updating code snippet

### ISSUE THREAD

**Issue by boghyon** [SAP/openui5-docs#19]
In the **SAP**UI5 version of the topic _[Versioning of SAPUI5](https://ui5.sap.com/#/topic/91f021426f4d1014b6dd926db0e91070)_, it's still mentioning the **deprecated** [`sap.ui.getVersionInfo`](https://openui5.hana.ondemand.com/api/sap.ui#methods/sap.ui.getVersionInfo).

> To access the SAPUI5 version at runtime, you can use the following code:
> ```
>var oPromise = sap.ui.getVersionInfo({ async: true });
>oPromise.then( function(oVersionInfo) { 
>var oVersion = jQuery.sap.Version(oVersionInfo.version);
>});
>```

Instead, it should be something like this:

```js
sap.ui.require([
    "sap/ui/VersionInfo",
    "sap/base/util/Version"
], function(VersionInfo, VersionUtil) {
    VersionInfo.load().then(function(oCurrentVersionInfo) {
        var oVersionUtil = new VersionUtil(oCurrentVersionInfo.version);
        // ...
    });
});
```

Additionally, I believe the above snippet could be also beneficial for OpenUI5 to be mentioned in the topic, not just for SAPUI5, in order to make use of the `sap/base/util/Version` module in the runtime.

**Comment by OlMue** [comment]
@codeworrior Can you confirm, if so, I will make the change. Thanks!

**Comment by codeworrior** [comment]
@OIMue I can confirm the first part (snippet is outdated, it should use the newer modules as sketched above). 

But I didn't get the rational behind the last proposal. To retrieve the OpenUI5 version, one doesn't have to require the `sap/base/util/Version` module. `oConfiguration.getVersion()` internally already does this. For sure, one can use the same snippet also with OpenUI5, but the version won't differ from the version retrieved from the `oConfiguration.getVersion()` call. 

@boghyon: if you could explain the last idea a bit more? Or did you want to make devs aware of the `sap/ui/VersionInfo` module and the info that it exposes, not `sap/base/util/Version`?

**Comment by boghyon** [comment]
Mainly, I was just requesting to update the outdated snippet in that SAPUI5 topic. And I couldn't think of any reasons why the same snippet wouldn't be useful to OpenUI5 users since `sap/base/util/Version` provides topic related APIs such as `compareTo`, `getMajor`, `getMinor`, `getPatch` which could be helpful if the app is dealing with versions in the runtime.

But if the purpose of that topic in OpenUI5 was just to inform how to retrieve the core version only, then `oConfig.getVersion()` (or even `sap.ui.version`?) is definitely enough.

**Comment by codeworrior** [comment]
`oConfig.getVersion()` returns an instance of `sap/base/util/Version`?

**Comment by boghyon** [comment]
Ooh, I see now. I was quite confused. :P Sorry for wasting time. Only the outdated snipped should be updated then. Please ignore the last proposal.

**Comment by OlMue** [comment]
Hi all, thanks for your feedback! I replaced the snippet. The updated documentation will be available as of 1.74.
